### ISSUE TITLE
An option to add newer Vitashell, Enso, Modoru and others apps like Autoplugin

### ISSUE THREAD

**Issue by leosel** [soarqin/finalhe#93]
An option to add newer Vitashell, Enso, Modoru and others apps like Autoplugin in the FinalHE tool

**Comment by leosel** [comment]
Is it possible?

**Comment by AntHJ** [comment]
Yes its possible as ive just made an All-In-One FinalHE...
this isnt a tutorial but is a guide to how I did it..

- Install the VPK to your Vita
- use QCMA to backup the App
- use Psvimgtools Easy FrontEnd to Extract the App
- rearrange the files in the App, Appmeta and License folders, for example move you need to move the folder
app\files\ux0_temp_game_PKGJ00000_app_PKGJ00000  into  app\ux0_temp_game_PKGJ00000_app_PKGJ00000
- Delete the now empty 'files' folders in each and the metadata file in each
- Create a folder with APP ID .. example PKGJ00000    <-- in CAPS 
- move the sce_sys folder into the APPID folder you just created
- highlight all the folders and use Winzip / 7zip / Winrar etc to zip the files

if you did this correctly then the resulting .Zip can now be places alongside the FinalHE app and will be detected when you next run FinalHE.. it will take a while to prep and transfer but its great having everything pre installable..

**Comment by AntHJ** [comment]
Not sure if this is allowed but this is my All in One pack
Vitashell, Modoru with 3.60/3.65 preloaded, Enso for 3.60/3.65, Autoplugin2, Account Switcher, IMCUnlock and more...

https://mega.nz/file/PwhVxSzS#CDq8gUx0Gw84WzAEKFjtAL99T9ecne168TZzZPEfKtA

**Comment by leosel** [comment]
Wow that's great, Thank you so much for the help,

**Comment by Babbosh** [comment]
> Vitashell, Modoru with 3.60/3.65 preloaded, Enso for 3.60/3.65, Autoplugin2, Account Switcher, IMCUnlock

Hi could you reupload. would really need this asap. Link is down now.

**Comment by theAkito** [comment]
> > Vitashell, Modoru with 3.60/3.65 preloaded, Enso for 3.60/3.65, Autoplugin2, Account Switcher, IMCUnlock
> 
> Hi could you reupload. would really need this asap. Link is down now.

Instead of doing the same mistake as last time, i.e. unreliably upload it, it should be distributed among as many peers as possible and then make those peers expose it for anyone to download, to introduce sufficient redundancy.

**Comment by Babbosh** [comment]
> > > Vitashell, Modoru with 3.60/3.65 preloaded, Enso for 3.60/3.65, Autoplugin2, Account Switcher, IMCUnlock
> > 
> > 
> > Hi could you reupload. would really need this asap. Link is down now.
> 
> Instead of doing the same mistake as last time, i.e. unreliably upload it, it should be distributed among as many peers as possible and then make those peers expose it for anyone to download, to introduce sufficient redundancy.

Yep. I can make a torrent out of it. I don't wanna buy a sony memory card every-time something goes wrong. Apparently the VS download error is quite common out of H-encore.

**Comment by AntHJ** [comment]
Is there a reason you need this?
Even tho its still fine to use, the FinalHE method is an old way of modding the Vita.

For the All in one FinalHE pack I did you can grab it from my github repos but to be honest your best just using the new Henlo method to mod and downgrade the system as its quicker and easier.

http://Jailbreak.psvita.group
http://Download.psvita.group

http://VideoGuide.psvita.group

**Comment by theAkito** [comment]
> Is there a reason you need this? Even tho its still fine to use, the FinalHE method is an old way of modding the Vita.
> 
> For the All in one FinalHE pack I did you can grab it from my github repos but to be honest your best just using the new Henlo method to mod and downgrade the system as its quicker and easier.
> 
> http://Jailbreak.psvita.group http://Download.psvita.group
> 
> http://VideoGuide.psvita.group

It's good to at least keep it for history's sake. 👍🏻
